packages/documentation/src/components/Header.jsx

```diff
@@ -37,17 +35,30 @@ class Search extends React.Component {
         // Map over each ID and return the full document
         .map(({ ref }) => this.index.documentStore.getDoc(ref)),
     });
-  };
+  }
 
   render() {
     return (
-      <div>
-        <input type="text" value={this.state.query} onChange={this.search} />
-        <ul>
+      <div id="search-container" className="site-search-container">
+        <i className="fas fa-search search-icon" />
+        <input
+          autoComplete="off"
+          type="search"
+          className="site-search-container__input"
+          id="search-input"
+          value={this.state.query}
+          onChange={this.search}
+          placeholder="Search..."
+          aria-label="search"
+        />
+        <ul
+          id="results-container"
+          className="site-search-results"
+          role="listbox"
+        >
           {this.state.results.map(page => (
             <li key={page.id}>
               <Link to={`/${page.path}`}>{page.title}</Link>
-              {`: ${page.tags.join(`,`)}`}
```

Comment:
Left the tags off for now. There's a lot we can do with this area to make the search nice. Would like some ideas- maybe we try to format it to look similar to the docusearch? @jbalboni @accbjt if you have other searches you like share them with me. 

I am going to add a lot of front matter to get the indexing working right. like I don't want people to have to type "Errorable" before seeing any of the components so I'll add "checkbox" to "ErrorableCheckbox"'s tags. I might break that off into a separate ticket so we can have a PR that's just focused on the indexing. 

![Screen Shot 2019-04-25 at 17 09 15](https://user-images.githubusercontent.com/4998130/56773883-6be12180-677d-11e9-8c6d-c46e9df00742.png)

Reply:
I'll limit this list size so it doesn't scroll